app/views/admin/categories/_form.html.erb

```diff
@@ -0,0 +1,14 @@
+<%= form_for ([:admin, @category]) do |f|  %>
+<%= render 'shared/error_messages',object: f.object %>
+
+    <div class= "form-group">
+        <%= f.label :title %>
+        <%= f.text_field :title,  class: "form-control" %>
+    </div>
+    <div class= "form-group">
+        <%= f.label :description %>
+        <%= f.text_field :description,  class: "form-control" %>
```

Comment:
@YutakaYamasaki Change `f.text_field :description` to `f.text_area :description`